.ci/install_kata_kernel.sh

```diff
@@ -26,6 +26,21 @@ build_and_install_kernel() {
 	
 	info "Installing '${kernel_type}' kernel"
 
+	case "${kernel_type}" in
```

Comment:
Not too long ago, we made a switch for the CI to use the tarballs built by kata-deploy so that scripts like `.ci/install_kata_kernel.sh` became just wrappers for kata-deploy. So if you need to add support to build the kernel for SNP you should do it in https://github.com/kata-containers/kata-containers/blob/CCv0/tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh first.

Reply:
@wainersm we have already added support for doing that. This part is being just for the tests. Since we did not have SNP test before, this part was not added.

Reply:
So those `extra_opts` options aren't needed, are they? @UnmeshDeodhar

Reply:
hi @UnmeshDeodhar , I still does not understand why it is needed.

Having tdx as an example, we got on https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh#L326 :

```
"-x tdx -u ${kernel_url}"
```

Reply:
@wainersm my understanding of this file was incorrect. Looks like there is another function called `build_and_install_kernel_for_cc` which will be used for building SEV and SNP kernel. Fixed this.